src/drivers/mct/cime_config/config_archive.xml

```diff
@@ -4,8 +4,8 @@
     <hist_file_extension>\.h.*.nc$</hist_file_extension>
     <rest_history_varname>unset</rest_history_varname>
     <rpointer>
-      <rpointer_file>rpointer.cpl</rpointer_file>
-      <rpointer_content >$CASE.cpl.r.$DATENAME.nc</rpointer_content>
+      <rpointer_file>rpointer$NINST_STRING.cpl</rpointer_file>
```

Comment:
Where is NINST_STRING set. What tests were done with archiving to ensure that this is working correctly in both the new and older implementations?